Watchman.AwsResources/Services/Sqs/QueueServiceData.cs

```diff
@@ -4,11 +4,13 @@
 
 namespace Watchman.AwsResources.Services.Sqs
 {
-    public class QueueServiceData
+
+    public class QueueDataV2
```

Comment:
renamed so it's clear it's old / new rather than QueueData vs QueueServiceData which I have to think about